src/grammar.lalrpop

```diff
@@ -175,19 +220,9 @@ Atom: RichTerm = {
     Bool => RichTerm::from(Term::Bool(<>)),
     StrChunks,
     Ident => RichTerm::from(Term::Var(<>)),
+    AsTerm<UniRecord>,
```

Comment:
I assume that at the end of the refactoring, you will only have a `Term: Richterm = AsTerm<TheWholeSyntax>`, correct?

Reply:
In the long term, yes. My current plan is however to try to just do that for records (more or less the only part that is "common" between terms and types), and otherwise get away by doing the translation on the fly, which would most of those definitions additionally to `AsTerm<>`. The goal is to avoid to have to implement a separate, mostly duplicate AST definition for the uniterm syntax before first release.